src/shared/components/Svg/Svg.tsx

```diff
@@ -1,20 +1,7 @@
 import { SvgProps } from './Svg.types';
 
-export const Svg = ({
-  id,
-  width = 24,
-  height = 24,
-  fill,
-  className,
-  'data-testid': dataTestid,
-}: SvgProps) => (
-  <svg
-    className={`svg-${id} ${className || ''}`}
-    width={width}
-    height={height}
-    fill={fill}
-    data-testid={dataTestid}
-  >
+export const Svg = ({ id, width = 24, height = 24, className, ...rest }: SvgProps) => (
+  <svg className={`svg-${id} ${className}`} width={width} height={height} {...rest}>
```

Comment:
Won't this cause `undefined` to be appended to the end of all the classnames that don't provide an explicit one?
```suggestion
  <svg className={`svg-${id} ${className || ''}`} width={width} height={height} {...rest}>
```